packages/zero-cache/src/services/mutagen/pusher.ts

```diff
@@ -324,6 +321,7 @@ class PushWorker {
         }
       }
     } else {
+      // Look for mutations results that should cause us to terminate the connection
       const groupedMutations = groupBy(response.mutations, m => m.id.clientID);
```

Comment:
we still loop through the API server's mutation responses as an `ooo mutation` should cause us to fatal the connection.